UI/update/sparkle-updater.mm

```diff
@@ -0,0 +1,105 @@
+#include "mac-update.hpp"
+
+#include <qaction.h>
+
+#import <Cocoa/Cocoa.h>
+#import <Sparkle/Sparkle.h>
+
+#ifdef __aarch64__
+#define BRANCH_SUFFIX "_arm64"
```

Comment:
Don't use macros for constants, use _actual_ constants for this:

```
NSString *const OBSBranchSuffix = @"_arm64";
```

We really need to stop using macros and macro constants as much as possible in our codebase moving forward.